iothub/service/src/Common/Exceptions/QuotaExceededException.cs

```diff
@@ -14,6 +14,11 @@ public QuotaExceededException(string message)
         {
         }
 
+        public QuotaExceededException(ErrorCode code, string message)
+            : base(code, message, true)
```

Comment:
I would add named parameter for `isTransient: true`. The `true` sitting in there as is can get a bit confusing.